sdk/java/build.gradle

```diff
@@ -35,12 +35,15 @@ if (file("${rootDir}/publish-mavencentral.gradle").exists()) {
 }
 
 dependencies {
-    implementation 'ai.picovoice:porcupine-java:1.9.3'
-    implementation 'ai.picovoice:rhino-java:1.6.3'
+//    implementation 'ai.picovoice:porcupine-java:2.0.0'
+//    implementation 'ai.picovoice:rhino-java:2.0.0'
+    implementation files('../../resources/rhino/binding/java/build/libs/Rhino.jar')
+    implementation files('../../resources/porcupine/binding/java/build/libs/Porcupine.jar')
     testImplementation 'org.junit.jupiter:junit-jupiter:5.4.2'
 }
 
 test {
+    systemProperty 'pvTestingAccessKey', System.getProperty('pvTestingAccessKey')
```

Comment:
why we need it here?

Reply:
to get accesskey for github action i guess